File: Problem1.php
<?php
$fizzbuzzCount = 0;
for ($i=0 ; $i < 1000; $i++) {
    if ($i % 3 == 0 | $i % 5 == 0) {
        $fizzbuzzCount = $fizzbuzzCount + $i;
    }
}
echo $fizzbuzzCount . "\r\n";
?>

File: Problem1.java
public class Problem1 {

    public static void main(String[] args)
    {
        int fizzbuzzCount = 0;
        
        for(int i=0; i<1000; i++)
        {
            if(i%3 == 0 || i%5 == 0)
            {
                fizzbuzzCount += i;
            }
        }
        System.out.println(fizzbuzzCount);
    }
}